assets/css/style.css

```
@@ -0,0 +1,72 @@
+body{
+  background-color: #f7f7f7;
+}
+
+.main_section{
+  display: flex;
+  margin: 0 auto;
+
+  width: 810px;
+
+  background: white;
+}
+
+.my_sidebar{
```

apply this for all classes
```suggestion
.my_sidebar {
```